src/main/java/seedu/address/model/Model.java

```diff
@@ -121,7 +125,7 @@ public interface Model {
      */
     void updateFilteredPersonList(Predicate<Person> predicate);
 
-    /**
```

Comment:
Is this a normal comment or JavaDoc comment?

Reply:
rectified in latest push